src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
+a {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  letter-spacing: normal;
+
+  --color-lightblue: #00acdc;
+}
+
+.product-card {
+  padding-top: 29px;
+  padding-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.image {
+  display: block;
+  height: 134px;
+  width: 160px;
+}
+
+.product-card__description {
+  margin-top: 45px;
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 13px;
+  width: 170px;
+}
+
+.description__title {
+  margin-bottom: 6px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.description__rating {
+  padding-top: 15px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
```

Comment:
```suggestion
```
'flex-direction: row;' is a default value. There's no need to write it for each of your flex-elements. they don't inherit flex-direction property from the container